kubernetes/migration/migrate_test.go

```diff
@@ -253,3 +252,27 @@ stubzone.org:53 {
 		}
 	}
 }
+
+func TestValidateVersions(t *testing.T) {
+	testCases := []struct {
+		from   string
+		to     string
+		shouldErr    bool
+	}{
+		{"1.3.1", "1.5.0", false},
+		{"banana", "1.5.0", true},
+		{"1.3.1", "apple", true},
+		{"banana", "apple", true},
```

Comment:
Maybe a couple more test cases where coredns to version <= coredns from version (1.4.0->1.3.1)